securedrop/journalist_app/admin.py

```diff
@@ -172,6 +176,24 @@ def edit_user(user_id):
                 else:
                     user.username = new_username
 
+            try:
+                first_name = request.form['first_name']
+                Journalist.check_name_acceptable(first_name)
+                if first_name != user.first_name:
+                    user.first_name = first_name
+            except FirstOrLastNameError as e:
+                flash('Invalid name: ' + str(e), 'error')
```

Comment:
These flashes need to be wrapped with a `gettext` and use a format string like `Invalid name: {name}`. This is done because some languages might not use a colon and want to reorder things.